package.json

```diff
@@ -12,6 +12,10 @@
         "branch": "",
         "SHA": ""
     },
+    "dependencies": {
```

Comment:
Wait, it might be a problem because ESLint has a lot of dependencies which are not present in its repository.